katsdpimager/report.py

```diff
@@ -345,6 +379,109 @@ def make_plots(self) -> Dict[str, bokeh.model.Model]:
             'parallactic_angle': self.make_plot_parallactic_angle(time_source)
         }
 
+    def make_uv_coverage(self) -> None:
+        """Generate SVG showing UV coverage and store it internally.
+
+        This is implemented in matplotlib (generating an SVG) rather than
+        Bokeh because Bokeh doesn't support elliptic arcs.
+        """
+        matplotlib.use('SVG')
+        fig = matplotlib.figure.Figure(figsize=(14, 14))
```

Comment:
Any reason for this figure size? I just had fun avoiding scaled images in my talk, so curious :-)

Reply:
The SVG image in the report scales to fill the width, but the original figure size ends up determining things like font sizes, line thicknesses etc. I played with this number a bit until it looked reasonable.

Reply:
Sounds familiar!